lib/ClangImporter/ImportMacro.cpp

```diff
@@ -381,7 +388,9 @@ static ValueDecl *importMacro(ClangImporter::Implementation &impl,
 
     // If it's a literal token, we might be able to translate the literal.
     if (tok.isLiteral()) {
-      return importLiteral(impl, DC, macro, name, tok, ClangN, castType);
+      return importLiteral(
+          impl, DC, macro, name, tok, ClangN, castType,
+          isOriginalMacroFunctionLike || macro->isFunctionLike());
```

Comment:
Absent any recursion, `importLiteral` is called immediately. Since `isOriginalMacroFunctionLike` is `false` by default, this means that `#define FOO() 1` would be imported as a global variable, not a Swift function.